### PR TITLE
Updated diagrams and docs on scheduling to reflect CICD changes

### LINKED ISSUE #151: Update Scheduling Document to reflect changes from Task CICD

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #151

### CODE REVIEW

File: content/documentation/pages/5-feature-guides/2-batch/2-scheduling.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ In this section, we will show you how you can schedule the launch of the batch j
 Spring Cloud Data Flow lets a user schedule the launching of tasks via a cron expression. A schedule can be created through the RESTful API or the Spring Cloud Data Flow UI.
 Spring Cloud Data Flow will schedule the execution of its tasks via a scheduling agent that is available on the cloud platform.
 ![Scheduling Architecture](images/SCDF-scheduling-architecture.png)
+The way that Spring Cloud Data Flow does this is by deploying a SchedulerTaskLauncher application to the platform and then
```
Comment (sabbyanandan) [inline content/documentation/pages/5-feature-guides/2-batch/2-scheduling.md]: Could we apply the same suggestions from the microsite PR?

Comment (sabbyanandan) [reply content/documentation/pages/5-feature-guides/2-batch/2-scheduling.md]: That'd be adding the hint re: cronjob and then the direct GH link to `SchedulerTaskLauncher` class.

Comment (cppwfs) [reply content/documentation/pages/5-feature-guides/2-batch/2-scheduling.md]: Updated based on code review!   Thanks!

**Comment by cppwfs** [comment]
Rebased Squashed Merged

**Pull request closed without merging**